scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -102,6 +96,21 @@
 
     protected Canvas visuPane;
 
+    private static final Logger LOGGER = Logger.getLogger(SchedulerPage.class.getName());
+
+    /** Actions on the scheduler */
+    private final static String START = "Start";
+
+    private final static String STOP = "Stop";
+
+    private final static String FREEZE = "Freeze";
+
+    private final static String RESUME = "Resume";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)